Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_tools.yml

```
@@ -167,20 +170,65 @@
-      Quantity: 2
+      Quantity: 1
+  - type: Hypospray
+    onlyAffectsMobs: true
+    injectOnly: true
 
+- type: entity
+  parent: HypoBorgStandard
+  id: HypoBorgMedicalAdvanced
+  name: advanced medical robot hypospray
+  description: A cyborg hypospray that can switch between a wide range of advanced medical chemicals.
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)